docs/admin/runai-setup/self-hosted/k8s/preparations.md

```diff
@@ -94,6 +95,57 @@ kubectl label node <NODE-NAME> node-role.kubernetes.io/runai-system=true
 !!! Warning
     Do not select the Kubernetes master as a `runai-system` node. This may cause Kubernetes to stop working (specifically if Kubernetes API Server is configured on 443 instead of the default 6443).
 
+### External Postgres database (optional)
+
+If you have opted to use an [external PostgreSQL database](prerequisites.md#external-postgresql-database-optional).
+, you need to perform initial setup to ensure successful installation. Follow these steps:
+
+1. Create a SQL script file, edit the parameters below, and save it locally:
```

Comment:
Maybe something like:
1. Create a SQL script file, edit the parameters below, and save it locally:
    * Replace `<DATABASE_NAME>` with a dedicate database name for RunAi  in your PostgreSQL database.
    * Replace `<ROLE_NAME>` with a dedicated  role name (user)  for RunAi database.
    * Replace `<ROLE_PASSWORD>` with a password for the new PostgreSQL role.
    * Replace `<GRAFANA_PASSWORD>` with the password to be set for Grafana integration.